### DISCUSSION TITLE
How to add custom sections to the docx report

### DISCUSSION THREAD

**Discussion by UserZer00** [pwndoc/pwndoc#314] [Q&A] [answered]
Hi all

I'm trying to add a section to the report that contains an executive summary. But somehow the generated report section stays blank after creating. 

That's the added section to the .docx template: 
![grafik](https://user-images.githubusercontent.com/87930535/170717210-882ea527-0595-4058-84fe-25e7fbd88773.png)

On the "Custom Sections" tab I added the following: 
![grafik](https://user-images.githubusercontent.com/87930535/170717362-dd53a6f5-3a66-4aad-bc4f-be23ae58d7da.png)

And on the "Custom Fields" tab I added this: 
![grafik](https://user-images.githubusercontent.com/87930535/170717509-1f1c27f9-b477-40c1-aa09-abab378f651b.png)

The new section shows up on the Audits tab and is editable. But as said, after generating the report it's blank.

I'd really appreciate some help. 

Best regards 
Dave

**Comment by UserZer00** [comment, marked as answer]
Nevermind - found the issue. The problem was that the "custom field" **didn't have the value "Text" as the Label**.

**Comment by mjrod1228** [comment]
I'm having the exact same issue and I'm starting to doubt if the Executive Summary is auto populated based on the Audit template.  Every time I download a report the Executive Summary is empty.

Here's a few screen shots of what I have.  Do you see anything that jumps out at you?    


![image](https://github.com/pwndoc/pwndoc/assets/120058122/9b09cc31-4b14-4066-91a3-5427ee1f397d)

![image](https://github.com/pwndoc/pwndoc/assets/120058122/cd2d3ac0-2919-43fc-bc58-98c3a5cca4c7)
